Userland/Utilities/headless-browser.cpp

```diff
@@ -661,12 +661,14 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     StringView resources_folder;
     StringView error_page_url;
     StringView ca_certs_path;
+    StringView screenshot_output_path = "output.png"sv;
```

Comment:
This should default to something with a timestamp so that a second default invocation does not overwrite the first screenshot

Reply:
Like here:

https://github.com/SerenityOS/serenity/blob/a7ce0509f7ebc582fbc65502e29b90833a1afa9e/Userland/Utilities/shot.cpp#L113

Reply:
Ok, i've done it now